### DISCUSSION TITLE
Windows Path not importing to bash $PATH correctly

### DISCUSSION THREAD

**Discussion by jbrzozo24** [microsoft/WSL#6566] [General]
Running ubuntu20.04 on WSL2 here. Whenever I launch ubuntu I get a slew of error messages such as 

"-bash: export: `Files/Intel/Intel(R)': not a valid identifier"
There is one for every component of my windows path. I've seen many issues here in trying to remove the windows path, but I would like to keep it. My windows path used to work, how could this have gotten broken? Any pointers are appreciated!

**Comment by benhillis** [comment]
Something is not handling the spaces in the path correctly, do you have any ideas what it might be?  What shell are you using?
